src/tags/forms.py

```diff
@@ -22,6 +23,11 @@ class Meta:
         model = Tag
         fields = ['name']
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['name'].validators = [validate_slug]
+        self.fields['name'].help_text = 'Tags cannot contain spaces, use hyphens to create multi-word-tags-like-this'
+
     def save(self, *args):
```

Comment:
I think this method can be deleted now.

Reply:
This save method override is actually necessary to properly update tags - it has no bearing on creation, but updating a tag's name value doesn't actually change its slug... Deleting this override seems to have no bearing on function until you assign the updated tag to an object, which will actually create a NEW tag using the OLD tag's name - demonstrated here:
![image](https://github.com/bytedeck/bytedeck/assets/105619909/40ec5fbb-6d86-45a2-a12e-c49c8a79dd52)
![image](https://github.com/bytedeck/bytedeck/assets/105619909/5d56d452-34c5-460f-8724-78cadeea6bcd)
![image](https://github.com/bytedeck/bytedeck/assets/105619909/1875d3cf-ce8a-49e6-9ec3-a72de3e607df)